src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FFIDMetadataRouteSpec.scala

```diff
@@ -101,12 +101,14 @@ class FFIDMetadataRouteSpec extends AnyFlatSpec with Matchers with TestRequest w
   private def resetDatabase(): Unit = {
     DbConnection.db.source.createConnection().prepareStatement("delete from FFIDMetadataMatches").executeUpdate()
     DbConnection.db.source.createConnection().prepareStatement("delete from FFIDMetadata").executeUpdate()
+    DbConnection.db.source.createConnection().prepareStatement("delete from FileMetadata").executeUpdate()
```

Comment:
Upper case commands "DELETE" and "FROM" for consistency?

Reply:
And these ones.